src/domains.js

```diff
@@ -105,6 +105,10 @@ export function getOrderAPIUrl() : string {
 }
 
 export function getPayPalDomainRegex() : RegExp {
+    if (__ENV__ === ENV.LOCAL) {
+        // eslint-disable-next-line security/detect-non-literal-regexp
+        return new RegExp(`${ getProtocol() }://${ getHost() }`);
```

Comment:
There might be problems here since `getHost()` contains special characters like `.` which will be turned into a regex. Can we just add localhost to the existing regex?

Reply:
I have us supporting domains with ngrok, localhost or loca.  Will add all.

Reply:
Maybe keep the if condition so that they aren't proxied in production to some malicious person